tests/generate_test.py

```diff
@@ -9,21 +9,23 @@
 
 def _get_test_case(name: str) -> Dict[str, Any]:
```

Comment:
```suggestion
def _get_test_case(name: str) -> Tuple[Dict[str, Any], List[str]]:
```
Along with adding `Tuple` at line 3
```
from typing import Any, Dict, List, Tuple
```

Reply:
Ah, thanks - I don't use type hints much so did my best to go with the flow :-)  I'll fix

Reply:
Type hints + mypy saved us from a lot of issues on a bigger project (but they can sometimes be a pain :P)

Reply:
Should be all set now